my-app/src/shared/components/ViewedVideos.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+
+export const ViewedVideos = ({ showViewed, viewAgain }) => {
+  if (showViewed.length > 5) {
+    return showViewed.shift();
```

Comment:
you are mutating props which is a side effect, please never do that
<img width="696" alt="Screenshot 2022-08-03 at 10 48 45" src="https://user-images.githubusercontent.com/57360190/182553598-5633a15d-0294-4c2a-9a23-8bece4044e3c.png">